### DISCUSSION TITLE
Using GitLens+ with a GitKraken Workspace using Pro Plan

### DISCUSSION THREAD

**Discussion by Guillaumeeee** [gitkraken/vscode-gitlens#2398] [Q&A]
Hi,

I was wondering, If I can use Git Lens + with my personal account on a workspace of my team that enabled Git Kraken Pro ?

In VS Code, I still get the windows that I use a trial version on Git Lens.

I am connected on the 2 platform, Git Kraken and Git Lens in VS Code, with the same github account.

**Comment by jkelroy** [comment]
Hello @Guillaumeeee, GitLens is free to work with any type of repository. [GitLens+ features](https://help.gitkraken.com/gitlens/gitlens-plus/) on local and public repositories can also be used for free. If the repositories are private, it would be required to have a [paid GitLens account](https://www.gitkraken.com/gitlens/pricing) to work with GitLens+ features. If you have additional questions, please let me know.
